python/baseline/tf/lm/model.py

```diff
@@ -222,6 +199,18 @@ def predict(self, batch_dict):
         step_softmax = self.sess.run(self.probs, feed_dict)
         return step_softmax
 
+    def _create_model_state(self, **kwargs):
+        embeddings_info = {}
+        for k, v in self.embeddings.items():
+            embeddings_info[k] = v.__class__.__name__
+        sess = kwargs.pop('sess', tf.Session())
```

Comment:
How does reloading wrok with multigpu? It just reloads a single model or will it be able to reload and redistribute to gpus?

Do you also need to pop the `'id'` key here?